Automattic/simplenote-macos
# Relocating Search Bar

### Details
In this PR we're relocating the Search Bar back into the Notes List.

cc @eshurakov Ready for 👀 sir!! Thanks in advance!!
cc @SylvesterWilmott 

Ref. #626

### Test: Searching
- [x] Verify entering any keyword in the Search Bar yields global results
- [x] Verify the Minimap reflects search results
- [x] Verify the Editor's Keyword Highlight works super
- [x] Verify clicking over the Editor when Search is active causes the Highlights to be dropped
- [x] Verify clearing the Search Bar's contents causes the Notes List to display the documents associated to the current selection

### Test: Dismissal
- [x] Verify clicking over any Tag / Trash causes the Search to be dismissed
- [x] Verify that clicking over the `+` New Note button causes the Search to be dismissed

### Test: Logout
1. Enter any random keyword in the search field
2. Logout
3. Log in again

- [x] Verify the Search Field is cleared out

### Release
`RELEASE-NOTES.txt` was updated in b08d1665dc276da8e2a640440549b10b0d57df21 with:

> SearchBar has been relocated into the Notes List #838
